<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMainWindow</class>
 <widget class="QMainWindow" name="NewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JahshakaVR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/app/icons/logo03.svg</normaloff>:/app/icons/logo03.svg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
*=======================================================================================
* Defaut StyleSheet for All System WIdgets and Components
* @Styled: Orane Campbell
* JahshakaVR 1.0.0
* Date: 10/27/2016
*=======================================================================================
*/

QTabWidget::pane{
    border: 1px solid #333333; top : -1px;
	background:rgb(67, 67, 67);
}

QTabWidget::tab-bar {
	background-color: rgb(54, 54, 54);
   /* border-bottom: 1px soild #434343;*/
}

QTabBar::tab {
	background-color:rgb(54, 54, 54);
 	border-bottom: 1px soild #333333;
    min-width: 15ex;
    min-height: 4ex;
    padding: 6px;
    min-width: 25ex;
}

QTabBar::tab:selected, QTabBar::tab:hover{
    border: 1px solid #333333; bottom : -1px;
	background-color:rgb(67, 67, 67);
    border-bottom: 0px;
    min-width: 8ex;
    padding: 5px;
    min-width: 20ex;
}

QTabBar::tab:selected {
    border: 1px solid #333333; bottom : -1px;
	background-color:rgb(67, 67, 67);
    border-bottom: 0px;
    min-width: 8ex;
    padding: 5px;
    min-width: 20ex;
}

QTabBar::tab:!selected {
    margin-top: 2px; 
}

QDockWidget {
	background-color: rgb(95, 95, 95);
    border: 1px solid lightgray;
    titlebar-close-icon: url(:/app/icons/delete.png);
    titlebar-normal-icon: url(:/app/icons/expand-button.png);
	color: rgb(255, 255, 255);
}

QDockWidget::title {
	height: 100px;
    text-align: center; /* align the text to the left */
    background: lightgray;
    padding-left: 5px 5px 5px 5pc;
	color: rgb(255, 255, 255);
	background-color: &quot;#282828&quot;;
}

QLabel {
color: rgb(197, 197, 197);
}

QTextEdit {
background-color: rgb(113, 113, 113);
color: rgb(172, 172, 172);
}

QSlider::groove:horizontal {
border: 1px solid #999999;
height: 30px;
}

QSlider::handle:horizontal {
width: 8px;
 /*background-image: url(:/slider-knob.png);*/
background: rgb(214, 214, 214);

}

QSlider::add-page:qlineargradient {
background: #4c4c4c;
border: 1px solid #4d4d4d;

}

QSlider::sub-page:qlineargradient{
background: #4c4c4c;
border: 1px solid #4d4d4d;
}

QSlider::handle:horizontal:hover {
background-color: rgb(30, 144, 255);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="MainSceneArea">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="sceneview">
       <attribute name="title">
        <string>Scene View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="sceneContainer" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: rgb(83, 83, 83);
    padding: 5px 5px 0px 5px;
	border: 1px soild black;
	opacity: 0.5;
}

QMenuBar::item {
    spacing: 6px; /* spacing between menu bar items */
    padding: 2px 6px;
    background: transparent;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #1e90ff;
}

QMenuBar::item:pressed {
    background: #1e90ff;
}


/* menu edit */

QMenu {
    border: 1px solid black;
    min-width: 200px;
}
/*
QMenu::item {
    background-color: transparent;
    min-width: 200px;
}

QMenu::item:selected {
    background-color: #1e90ff;
    border-bottom: 1px solid #1eb2ff;
}

QMenu::item:selected {
    border-color: darkblue;
    background: #1e90ff;
    border-bottom: 1px solid #1eb2ff;
}
</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/app/icons/file-in-folder.svg</normaloff>:/app/icons/file-in-folder.svg</iconset>
     </property>
     <addaction name="actionBlank"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionVisit_Website"/>
    <addaction name="actionBlog"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="sceneHierarchy">
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>564</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>524287</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Scene Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="sceneHierarchyContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="LayerTreeWidget" name="sceneTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>210</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QTreeView::item:hover {
    background: #42b5e8;
}

QTreeView::item:selected {
    background: #1e90ff;
}

QTreeView::item:selected:active{
    background: #1e90ff;
}

QTreeView::item:selected:!active {
    background: #98dbf8;
}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        image: url(&quot;:/icons/keyboard-right-arrow-button.svg&quot;);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        image: url(&quot;:/icons/expand-button.svg&quot;);
}</string>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="sceneTreeContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: white;
    background-color: rgb(113, 113, 113);
    border-radius: 2px;
/*
    padding: -25px 0px 5px 0;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:icons/add.svg&quot;);*/
}	

QPushButton:hover {
    color: white;
	background-color: rgb(0, 170, 255);
    border-radius: 2px;
/*
    padding: -25px 0px 5px 0;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:icons/add.svg&quot;);*/
}

</string>
          </property>
          <property name="text">
           <string>Remove Object</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
    color: white;
    background-color: rgb(113, 113, 113);
    border-radius: 2px;
/*
    padding: -25px 0px 5px 0;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:icons/add.svg&quot;);*/
}	

QToolButton:hover {
    color: white;
	background-color: rgb(0, 170, 255);
    border-radius: 2px;
/*
    padding: -25px 0px 5px 0;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:icons/add.svg&quot;);*/
}

</string>
          </property>
          <property name="text">
           <string>Add Object</string>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
    <zorder>sceneTreeContainer</zorder>
    <zorder>sceneTree</zorder>
    <zorder>widget</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ModelsPresetsDock">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>212</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>3D Model Presets</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="layout">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ModelPresets" name="modelpresets" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="AnimationDock">
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>250</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="layout_2">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="AnimationWidget" name="animationtimeline"/>
     </item>
     <item row="0" column="0">
      <widget class="MainTimelineWidget" name="mainTimeline" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="PropertiesDock">
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QScrollArea" name="Propertieswidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>220</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="accordian">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>298</width>
          <height>270</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ObjectViewDock">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>160</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Object View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/save-file-button.svg</normaloff>:/app/icons/save-file-button.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/open-file-button.svg</normaloff>:/app/icons/open-file-button.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEditorCamera">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/video-camera.svg</normaloff>:/app/icons/video-camera.svg</iconset>
   </property>
   <property name="text">
    <string>Editor Camera</string>
   </property>
  </action>
  <action name="actionViewerCamera">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/photo-camera.svg</normaloff>:/app/icons/photo-camera.svg</iconset>
   </property>
   <property name="text">
    <string>Viewer Camera</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/save-file-button.svg</normaloff>:/app/icons/save-file-button.svg</iconset>
   </property>
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/add-folder-button.svg</normaloff>:/app/icons/add-folder-button.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/folder-settings-button.svg</normaloff>:/app/icons/folder-settings-button.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/cancel.svg</normaloff>:/app/icons/cancel.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/businessman-talking-about-data-analysis.svg</normaloff>:/app/icons/businessman-talking-about-data-analysis.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/document.svg</normaloff>:/app/icons/document.svg</iconset>
   </property>
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionVisit_Website">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/responsive-website-design-on-monitor-screen.svg</normaloff>:/app/icons/responsive-website-design-on-monitor-screen.svg</iconset>
   </property>
   <property name="text">
    <string>Visit Website</string>
   </property>
  </action>
  <action name="actionBlog">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/responsive-website-design-on-monitor-screen.svg</normaloff>:/app/icons/responsive-website-design-on-monitor-screen.svg</iconset>
   </property>
   <property name="text">
    <string>Blog</string>
   </property>
  </action>
  <action name="actionBlank">
   <property name="text">
    <string>blank</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/undo.svg</normaloff>:/app/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app/icons/redo.svg</normaloff>:/app/icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainTimelineWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/widgets/maintimelinewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">src/widgets/layertreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>AnimationWidget</class>
   <extends>QFrame</extends>
   <header>src/widgets/animationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModelPresets</class>
   <extends>QWidget</extends>
   <header>src/widgets/modelpresets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
